debian/config

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+# There are two questions open:
+
+# wartool -m  is inconsistently documented. It can be easily added here, but is it still part of wartool?
```

Comment:
-m isn't actually used anymore, it should be removed from the wartool.cpp help text